SNOW-595309: Adjust README CONTRIBUTING and MANIFEST (#359)
snowflakedb/snowpark-python: README.md

```bash
pip install snowflake-snowpark-python
```

> Optionally, you may want to install pandas as well to the same environment if you use pandas related features:

Reviewer: ```suggestion
Optionally, you have to install pandas in the same environment if you want to use pandas-related features:
```

df = session.create_dataframe([[1, 2], [3, 4]], schema=["a", "b"])
df = df.filter(df.a > 1)
df.show()

> pandas_df = df.to_pandas()  # this requires pandas installed in the python environment

Reviewer: ```suggestion
pandas_df = df.to_pandas()  # this requires pandas installed in the Python environment
```

result = df.collect()
```

## Samples
The [Developer Guide][developer guide] and [API references][api references] have basic sample code.

> [Snowflake-Labs][snowflake lab sample code] has sophisticated demos.

Reviewer: ```suggestion
[Snowflake-Labs][snowflake lab sample code] has more curated examples of using snowpark for data analytics and data science.
```
